install/configuring_inventory_file.adoc

```diff
@@ -131,9 +131,15 @@ Configuration] for more information.
 |This variable overrides the host name for the cluster, which defaults to the
 host name of the master.
 
-|`openshift_master_cluster_public_hostname`
+a|`openshift_master_cluster_public_hostname`
 |This variable overrides the public host name for the cluster, which defaults to
-the host name of the master.
+the host name of the master. If an external load balancer is used public hostname should resolve to external load balancer address.
```

Comment:
Maybe "If you use an external load balancer, specify the external load balancer's address."